utilities/EluvioLiveCli.js

```diff
@@ -257,7 +257,7 @@ const CmdTenantShow = async ({ argv }) => {
   console.log("check_nfts", argv.check_nfts);
   try {
     await Init();
-
+    //FIXME:
```

Comment:
what does this fixme need fixed?

Reply:
Oops, forgot to remove my comment for a temp thing. Pushed fix.